### ISSUE TITLE
CapabilitiesHandler only accepts String capability values

### ISSUE THREAD

**Issue by cwensel** [bonigarcia/selenium-jupiter#409]
**Description of the problem**: 

when writing a template test, and `browsers.json` has a capability, it must be of type String. But when the capability is passed down, creating the remote driver fails as the value is expected to be a Boolean.

```
      {
        "type": "chrome",
        "capabilities": {
          "acceptInsecureCerts": "true"
        }
      }
```

if you change the type to boolean, the reflection invocation fails.

https://github.com/bonigarcia/selenium-jupiter/blob/a5c199dbefd5977f37b5b38e58bb9fa90b62f3a3/src/main/java/io/github/bonigarcia/seljup/CapabilitiesHandler.java#L179-L180

Note, support for both `boolean` and `map` value arguments would be welcome.

```
      {
        "type": "chrome",
        "capabilities": {
          "acceptInsecureCerts": true,
          "selenoid:options": {
            "enableVideo": true,
            "enableLog": true,
            "hostsEntries": "host.docker.internal:host-gateway"
          }
        }
      }
```

**Browser and version**: 

All browsers.

**Selenium-Jupiter version**: 

io.github.bonigarcia:selenium-jupiter:4.3.6

**Selenium-Jupiter traces**:

```
2023-09-12 14:44:30 [main] TRACE i.g.b.wdm.webdriver.WebDriverCreator.createRemoteWebDriver(86) -- Requesting http://localhost:56814/wd/hub (the response code is 200)
2023-09-12 14:44:31 [main] TRACE i.g.b.wdm.webdriver.WebDriverCreator.createRemoteWebDriver(96) -- SessionNotCreatedException creating WebDriver object (Could not start a new session. Response code 400. Message: invalid argument: entry 0 of 'firstMatch' is invalid
from invalid argument: cannot parse capability: acceptInsecureCerts
from invalid argument: must be a boolean 
```

**Error trace**:

**Comment by bonigarcia** [comment]
Thanks for reporting. I'll look into it as soon as I find some time.

**Comment by bonigarcia** [comment]
This should be fixed in Selenium-Jupiter 5.0.0, just released.
